plugins/dfu/meson.build

```diff
@@ -1,4 +1,4 @@
-if get_option('gusb')
+if get_option('gusb') and host_machine.system() == 'linux'
```

Comment:
Wait, why doesn't this work on win32?

Reply:
@superm1 It doesn't work on *BSDs. [G_USB_DEVICE_LANGID_ENGLISH_UNITED_STATES](https://github.com/fwupd/fwupd/blob/f3c64adb6c6d4ad6b2045682c48cdd3bbf3a85a7/plugins/dfu/fu-dfu-device.c#L1232) is undeclared. I will add win32 to this condition.

Reply:
Hmm, are you perhaps using too old of a gusb version?  That code seems pretty straightforward.
https://github.com/hughsie/libgusb/blob/master/gusb/gusb-device.h#L129

Reply:
Looks like 0.3.6 or newer is needed. https://github.com/hughsie/libgusb/commit/38d1f8b3af5e76b548b5701f39ac9dd8a74bde9e

Reply:
@superm1 My bad. I had old gusb and that was the problem.